integration_tests/power_flow_tests.feature

```diff
@@ -1,5 +1,6 @@
 Feature: Run power flow integration tests
 
+  @disabled
```

Comment:
This is temporary, right?

Reply:
The problem here is that power flow is controlled by a config, namely POWER_FLOW, and the Python interpreter (which is pointless as a distinction IMO, since there might be people that want to disable power flow, but still use Python interpreter). The fact that this is a config has the disadvantage that it is not controllable when running console integration tests (because there we do not have access to the config). Also, the default value of this is true, which is surprising considering that this feature is not complete. This leads to this integration test failing, because pandapower was removed as a dependency a while ago. 
This was some background on the issue, and the reason why I decided to ignore the mess and just disable the test. If we want to resolve this properly (which should probably happen in the context of the power flow subtasks) the power flow config needs to be moved to ConstantsLimits and this test needs to be converted to a non-console integration test, so that we can enable the power flow by changing the ConstantsLimits.

Reply:
Ok, thank you! Makes sense. Lets get to it when adding proper power flow.